LICENSE-BSD-3-Clause

```diff
@@ -0,0 +1,26 @@
+Copyright 2022 The rust-vmm authors.
```

Comment:
It looks like there was a commit in 2023 (besides from me and the robot) in case you are tracking copyright years.  Other crates under the org don't seem to do that, though.